docs/user-guide/_cli-using-team-configuration-application-developers.md

```diff
@@ -1,30 +1,40 @@
-# Team configuration for application developers
+# Configuration for application developers
 
 As an application developer or Zowe CLI user, you want to manage your connection details efficiently and in one location.
 
 That could mean relying on a team configuration file, or creating your own *user* configuration file.
 
 To create your own user configuration, start with a global team configuration file that you have created or was provided to you. In this way, a single global configuration can become the basis for multiple user-specific configurations that are created with modifications tailored to particular requirements.
```

Comment:
"... start with a global team configuration file that you have created or was provided to you."
to
"start with a global team configuration file that you created or was provided to you."